include/wil/winrt.h

```diff
@@ -30,14 +30,11 @@
 
 /// @cond
 #if defined(WIL_ENABLE_EXCEPTIONS) && !defined(__WI_HAS_STD_LESS)
-#ifdef __has_include
-#if __has_include(<functional>)
 #define __WI_HAS_STD_LESS 1
+#if WI_HAS_INCLUDE(<functional>, 1) // Functional header available since C++11... fall back to assume present

```

Comment:
Is there any point using include guards for something dating to cpp11?  That is well below the support floor so having guards at all seems unnecessary.

Reply:
We still need to account for OS code that may not even _have_ access to the STL, in which case a check like this one will fail.